src/main/java/com/cinema/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package com.cinema.controller;
+
+import com.cinema.model.dto.UserResponceDto;
+import com.cinema.service.UserService;
+import com.cinema.util.mapper.UserMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users/byemail")
```

Comment:
```suggestion
@RequestMapping("/users")
```
тут має вказуватися маппінг загальний для всіх методів